src/main/webapp/app/components/Tabs/tabs.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect } from 'react';
+import { Col, Container, Row } from 'reactstrap';
+import './tabs.scss';
+import { observer } from 'mobx-react-lite';
+import classNames from 'classnames';
+import { IRootStore } from 'app/stores/createStore';
+import { componentInject } from 'app/shared/util/typed-inject';
+
+export type Tab = {
+  title: string;
+  content: React.ReactNode;
+};
+
+export interface ITabsProps extends StoreProps {
+  tabs: Tab[];
+}
+
+const Tabs = observer(({ tabs, tabsStore }: ITabsProps) => {
```

Comment:
Why do we create our own Tabs component?

Reply:
I didn't realize reactstrap had its own Tabs component until now. This one wasn't too hard to create and isn't a very large file. I can try refactoring to use reactstrap if you think that would be less bloat.

Reply:
I will not create component if we can use existing library unless we have special needs.